### DISCUSSION TITLE
Is It Possible to Use A Higher LZ4 Compression Level?

### DISCUSSION THREAD

**Discussion by GeorgiaM-honestly** [abbbi/virtnbdbackup#75] [Q&A / Support]
Hello,
Is it possible to use a higher lz4 compression level? My destination is fairly slow so it makes sense in my use case to use heavier compression. I would like to use lz4's -8 option if possible.

**Comment by abbbi** [comment]


**Reply by abbbi** [reply]
Latest release supports adding compression level, see documentation.

**Reply by GeorgiaM-honestly** [reply]
Oh wow, fantastic!! Thanks so much! Works great here.
